templates/trivia_qa/rc/templates.yaml

```
@@ -0,0 +1,99 @@
+dataset: trivia_qa
+subset: rc
+templates:
+  014c221a-ed54-4a1d-bc67-2ded6ea8c528: !Template
+    id: 014c221a-ed54-4a1d-bc67-2ded6ea8c528
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nI've always wondered: {{question}}\
+      \ {{\"\\n\"}} Can you answer from wikipedia? You can try multiple times. |||\
+      \ {% for v in answer.aliases -%} Answer: {{v}} {% endfor -%}\n{% endif %}"
+    name: context_self_description_wikipedia_multiple_answer
+    reference: Ask question by self description from wikipedia
+    task_template: true
+  14acf45f-a958-4fff-a8fc-60a1f241059a: !Template
+    id: 14acf45f-a958-4fff-a8fc-60a1f241059a
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nCan you tell me the answer\
+      \ of the following question from wikipedia? You can guess multiple answers.\
+      \ {{\"\\n\"}} Question: {{question}} ||| {% for v in answer.aliases -%} Answer:\
+      \ {{v}} {% endfor -%}\n{% endif %}"
+    name: wikipedia_question_multiple_answer
+    reference: Mention the KB with multiple answer
+    task_template: true
+  1a4b6c15-de55-407e-8f45-53e55207c59d: !Template
+    id: 1a4b6c15-de55-407e-8f45-53e55207c59d
+    jinja: Answer the following question. Answers will be separated by "<SEP\>". {{"\n"}}
+      {{question}} ||| {%- for v in answer.aliases -%} {{v}} {{- " <SEP\> " if not
+      loop.last else "" -}} {%- endfor -%}
+    name: question_with_instruction_multiple_valid_answer
+    reference: 'Generate multiple valid answer from question with an instruction. '
+    task_template: true
+  2725b229-1217-43ed-98b0-32cf5c4293b3: !Template
+    id: 2725b229-1217-43ed-98b0-32cf5c4293b3
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nAccording to wikipedia,\
+      \ {{question}} ||| {{answer.aliases|choice}} \n{% endif %}"
+    name: wikipedia_contextual_question
+    reference: Ask question using wikipedia keyword
+    task_template: true
+  5946db1a-a068-4a31-a06f-74a7d976cb6d: !Template
+    id: 5946db1a-a068-4a31-a06f-74a7d976cb6d
+    jinja: Guess a question from the topic "{{answer.aliases|choice}} " ||| {{question}}
+    name: guess_question
+    reference: Guess a question. It will show if model can evaluate entity in question.
+    task_template: false
+  69b1d0d7-ee22-4313-a8a0-89b718177d2a: !Template
+    id: 69b1d0d7-ee22-4313-a8a0-89b718177d2a
+    jinja: "{% if answer.type == \"WikipediaEntity\" %} \nI''ve always wondered: {{question}}{{\"\
+      \\n\"}}\nCan you search wikipedia and answer? ||| {{answer.aliases|choice}}\
+      \ \n{% endif %}\n"
+    name: context_self_description_wikipedia
+    reference: Ask question by self description from wikipedia
+    task_template: true
```

it's a reading comprehension dataset
```suggestion
    task_template: false
```